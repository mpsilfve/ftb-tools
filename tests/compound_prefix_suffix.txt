etu- ja takapuoli tai -kolmannes - - kissa
