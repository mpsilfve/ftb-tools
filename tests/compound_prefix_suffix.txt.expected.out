
"<etu->"
	"etu" Trunc Prefix abc #1
"<ja>"
	"ja" Adv abc #1
	"ja" CC abc #1
"<takapuoli>"
	"takapuoli" N Sg Nom abc #1
"<tai>"
	"tai" CC abc #1
	"tai" N Sg Nom abc #1
"<-kolmannes>"
	"kolmannes" N Sg Nom Suffix abc #1
"<->"
	"-" Pun noabc #1
"<->"
	"-" Pun noabc #1
"<kissa>"
	"kissa" N Sg Nom abc #1
